cli/src/main/java/org/jboss/as/cli/operation/OperationRequestCompleter.java

```diff
@@ -492,6 +492,19 @@ protected int complete(CommandContext ctx, ParsedCommandLine parsedCmd, Operatio
         return parsedCmd.endsOnSeparator() ? parsedCmd.getLastSeparatorIndex() + 1 : parsedCmd.getLastChunkIndex();
     }
 
+    private boolean suggestionEqualsUserEntry(List<String> candidates, String chunk, int suggestionOffset) {
+        if (chunk == null) {
+            return false;
+        }
+
+        if (suggestionOffset > 0) {
+            // user entry before suggestionOffset is always the same - compare only part after offset
+            return candidates.size() == 1 && chunk.substring(suggestionOffset).equals(candidates.get(0));
```

Comment:
Hi @spyrkob,
excellent! Perhaps you could move the 2 candidates.size() == 1 to a single checkline 496.
if(chunk == null || candidates.size() == 1)